osquery/config/config.cpp

```diff
@@ -133,8 +133,7 @@ Status Config::getMD5(std::string& hashString) {
     return s;
   }
 
-  osquery::md5::MD5 digest;
-  hashString = std::string(digest.digestString(config_string.c_str()));
+  hashString = computeMD5((unsigned char *)hashString.c_str(), hashString.length());
```

Comment:
This is crufty old code, but do you mind changing `hashString` to `hash_string` while you're here?